doc/source/getting_started/docker/windows_container.rst

```diff
@@ -0,0 +1,165 @@
+.. _ref_windows_docker:
+
+Geometry Service as a Windows Docker container
+==============================================
+
+.. contents::
+
+.. _ref_running_windows_containers:
+
+Running Windows Docker containers
+---------------------------------
+
+To run the Windows Docker container for the Geometry service, ensure that you follow
+these steps when installing Docker:
+
+#. Install `Docker Desktop <https://docs.docker.com/desktop/install/windows-install/>`_.
+
+#. When prompted for **Use WSL2 instead of Hyper-V (recommended)**, **UNTICK** this checkbox.
+   Hyper-V must be enabled to run Windows Docker containers.
+
+#. Once the installation finishes, restart your machine, and start Docker Desktop.
+
+#. On the Windows taskbar, go to the **Show hidden icons** section, right-click in the Docker
+   Desktop app, and select **Switch to Windows containers**.
+
+Now that your Docker engine supports running Windows Docker containers, you can build or install
+the PyAnsys Geometry image.
+
+Build or install the Geometry Service image
+-------------------------------------------
+
+There are two options for users to install the PyAnsys Geometry image:
+
+* Downloading it from the :ref:`GitHub Container Registry <ref_windows_docker_ghcr>`.
+* :ref:`Building the Geometry Service Windows container <ref_windows_docker_fromscratch>`.
+
+.. _ref_windows_docker_ghcr:
+
+GitHub Container Registry
```

Comment:
Start this title with a simple verb? "Register the GitHub container"
Vale likely will flag the existing title as not using sentence case.